mig-loader/loader.go

```diff
@@ -346,6 +347,29 @@ func checkEntry(entry mig.BundleDictionaryEntry) (err error) {
 	return
 }
 
+// Check the file permissions set on a bundle entry and if they are
+// incorrect fix the permissions
+func entryPermCheck(entry mig.BundleDictionaryEntry) (err error) {
+	defer func() {
+		if e := recover(); e != nil {
+			err = fmt.Errorf("entryPermCheck() -> %v", e)
+		}
+	}()
+	var fi os.FileInfo
+	fi, err = os.Stat(entry.Path)
+	if err != nil {
+		panic(err)
+	}
+	if fi.Mode() != entry.Perm {
```

Comment:
is it faster to skip the `Stat` and `Mode` call and just do the `Chmod` regardless?